### DISCUSSION TITLE
addons su core + Raspiberry os

### DISCUSSION THREAD

**Discussion by gianlucadegiorgio** [indomus/forum#3273] [HUB: Home Assistant]
Salve a tutti e CHIEDO SCUSA se la domanda è troppo stupida o già fonte rdi risposta in passato. 
Molto semplicemente, ho installato HA core su raspberry OS ... l'installazione con venv per capirci. Ho capito che in questo tipo di installazione le cose vanni installate a "manella", ma faccio un esempio così ci capiamo meglio. Ho bisogno di installare NUT per il mio UPS. Chiaramente non ho gli add-on e l'unica cosa che trovo è l'integrazione lato client. Mi manca il lato server. Come faccio ad installarla ? Dove la trovo ? In generale per tutti gli add-on, che strada devo seguire ?
Grazie .. ed abbiate pazienza.

**Comment by snakuzzo** [comment]
Ti dico già che hai sbagliato tutto quanto.
L'installazione in venv è sostanzialmente per gli sviluppatori e/o persone che sanno dove mettere le mani. Non offenderti, ma se fai queste domande vuol dire che non sai assolutamente dove mettere le mani.
Il consiglio che posso darti è partire da HA OS poi, col tempo, deciderai se migrare a Core nudo e crudo. In tal caso, di sicuro ti sconsiglierei l'installazione in venv, ma userei docker.
A tal proposito, ti consiglio di leggere questo articolo -> https://indomus.it/guide/home-assistant-hassio-o-home-assistant-core-quale-installazione-fa-al-caso-mio/

Comunque, tornando alla tua domanda iniziale -> https://indomus.it/guide/gestire-un-ups-con-raspberry-pi-tramite-network-ups-tools-su-raspbian/

**Reply by gianlucadegiorgio** [reply]
Grazie per la risposta. Si forse potresti avere ragione. Tuttavia, i NUT, server e client li ho installati prima della tua risposta, configurati con drivers e tutto. Leggendo la documentazione che evidentemente alla fine riesco "stranamente" a comprendere. E forse, ma dico forse, le mani in fin dei conti so dove metterle. Era solo che, siccome sono uno che è consapevole dei propri limiti con ambienti ed applicativi che non conosco, volevo una conferma da chi, sicuramente come te, ha molta più esperienza da sviluppatore.
Grazie comuque per il consiglio ... passerò volentieri alla versione con docker.
